LondonBritishLibrary/orient/BLorient786.xml

```diff
@@ -0,0 +1,239 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BLorient786" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Rǝtuʿa Haymānot, Collection of homilies for various occasions</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DE"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <idno>Wright cat. CCCXLIII</idno>
+                    </msIdentifier>
+
+<msContents>
+
+
+  <msItem xml:id="ms_i1">
+  <locus from="5r"/>
+  <title type="complete" ref="LIT2222RetuaH">Rǝtuʿa Hāymānōt, or "the Orthodox", a collection of homilies for various occasions</title>
+  <textLang mainLang="gez"></textLang>
+</msItem>
```

Comment:
The closing tag should come at the very end, so that ms_1.1, ms_1.2 etc, are included into ms_i1.